cmd/installer/configureStorage.go

```diff
@@ -579,6 +575,8 @@ func remapDevice(device, target string) string {
 }
 
 func unmountStorage(logger log.DebugLogger) error {
+	syscall.Sync()
+	time.Sleep(time.Millisecond * 100)
```

Comment:
Why 100 ms? is there any better way to do this?

Reply:
It's just a guess based on experience. I don't know of a way to check for I/O activity to settle down.

Reply:
Add todo for checking against sys? https://www.kernel.org/doc/Documentation/block/stat.txt

Reply:
I think that's more complication than needed. Also, it's not just whether the kernel I/O is idle, but whether the storage device has actually committed to the media.
This is just a low-cost delay.